test/common/io/io_uring_impl_test.cc

```diff
@@ -29,7 +29,7 @@ class IoUringBaseTest : public ::testing::Test {
 };
 
 class IoUringImplParamTest
-    : public IoUringBaseTest,
+    : public IoUringImplTest,
```

Comment:
nit: merge  `IoUringImplParamTest` and `IoUringImplTest` (unless you plan to have more extensions.